src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -501,12 +327,12 @@ You can declare [custom redirects](https://developers.cloudflare.com/pages/platf
 
 ### Routes
 
-You can define which routes are invoking functions and which are static assets, using [Cloudflare routing](https://developers.cloudflare.com/pages/platform/functions/routing/#functions-invocation-routes) via a `_routes.json` file. This file is automatically generated by Astro.
+You can define which routes are invoking functions and which are static assets, using [Cloudflare routing](https://developers.cloudflare.com/pages/platform/functions/routing/#functions-invocation-routes) via a `_routes.json` file.
 
 #### Custom `_routes.json`
 
-By default, `@astrojs/cloudflare` will generate a `_routes.json` file with `include` and `exclude` rules based on your applications's dynamic and static routes.
-This will enable Cloudflare to serve files and process static redirects without a function invocation. Creating a custom `_routes.json` will override this automatic optimization. See [Cloudflare's documentation on creating a custom `routes.json`](https://developers.cloudflare.com/pages/platform/functions/routing/#create-a-_routesjson-file) for more details.
+By default, the adapter will generate a `_routes.json` file with `include` and `exclude` rules based on your applications.
```

Comment:
is this done using the `routes.extend` option mentioned above? is so maybe this could clearly reference that?